test/quantization/fx/test_quantize_fx.py

```diff
@@ -5619,39 +5660,68 @@ def forward(self, x):
         }
         self.checkGraphModuleNodes(m, expected_node_occurrence=node_occurrence)
 
-    @skipIfNoONEDNN
-    def test_linear_leaky_relu_lowering(self):
-        """ Test fusion and lowering of Linear - (bn -) LeakyReLU
-            by FX. For onednn backedn only.
-        """
-        from torch.ao.quantization.backend_config import get_onednn_backend_config
-        qconfig_mapping = get_default_qconfig_mapping('onednn')
+    def _test_activation_fusion_lowering_helper(
```

Comment:
nit: maybe `_test_linear_activation_fusion_lowering_helper`

Reply:
Ok. It's fixed.